nevergrad/benchmark/experiments.py

```diff
@@ -1255,7 +1261,7 @@ def gym_anm(
 @registry.register
 def compiler_gym(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
     """Working on CompilerGym."""
-    return gym_anm(seed, specific_problem="compilergym")
+    return gym_anm(seed, specific_problem="compilergym", pb_index=np.random.randint(23))
```

Comment:
Is there any need in using a random number generator `np.random.randint(23)` here, rather than just using `pb_index=seed % 23` and calling it with a range of 23 consecutive seeds?

Reply:
I do what you describe in "problems23_compiler_gym". The goal of "compiler_gym" is just to do a fast experiment for checking.